File: php/testcode/benchmark_test_00060.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest00060(BenchmarkRequest $req): BenchmarkResponse {
    $id = $req->param('id');
    if ((int)$id === 42) {
        return BenchmarkResponse::ok('found');
    }
    return BenchmarkResponse::badRequest('not found');
}

File: rust/testcode/benchmark_test_00060.rs
pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let body = req.param("body");
    let obj = deserialize_object(body.as_bytes());
    super::shared::BenchmarkResponse::ok(&format!("deserialized len={}", obj))
}

fn deserialize_object(_data: &[u8]) -> usize { 42 }
